lib/grape-swagger/endpoint.rb

```diff
@@ -264,6 +261,12 @@ def tag_object(route, path)
 
     private
 
+    def build_reference(route, value, response_model)
```

Comment:
Extracted this to a method to avoid lint:
```
lib/grape-swagger/endpoint.rb:196:5: C: Metrics/PerceivedComplexity: Perceived complexity for response_object is too high. [15/14]
```